omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -94,6 +93,16 @@ private void CreateApiDefinition() {
 		swaggerSpecification.setConsumes(consumes);
 	}
 	
+	private boolean TestOperationImplemented(OperationEnum operation, BaseDelegatingResource resource) {
+		Method method = null;
+		switch (operation) {
+			case get:
+				method = ReflectionUtils.findMethod(resource.getClass(), Operation.OPERATION_GET_ALL_METHOD, RequestContext.class);

```

Comment:
I doubt if this is guaranteed to work because a resource can implement a method and just throw an exception, this is why i would assume invoking the method and catching the ResourceDoesNotSupportOperationException is still the better option